### PR TITLE
Terminal Plugin API: by default, add a new line to the text being sent to the terminal

### PR DESCRIPTION
**Pull request by azatsarynnyy**
Signed-off-by: Artem Zatsarynnyi <azatsary@redhat.com>

This PR makes the `Terminal.sendText` API consistent with VSCode. If it's called with omitted second parameter (`addNewLine`) then a new line will be added to the text being sent to the terminal since this is normally required to run a command in the terminal.

It's needed for https://github.com/eclipse/che/issues/10574

### CODE REVIEW

File: packages/plugin-ext/src/plugin/terminal-ext.ts [hunk 1]
```diff
@@ -79,7 +79,7 @@ export class TerminalExtImpl implements Terminal {
         this.termProcessId = this.proxy.$createTerminal(nameOrOptions);
     }
 
-    sendText(text: string, addNewLine?: boolean): void {
+    sendText(text: string, addNewLine: boolean = true): void {
```
Comment (AndrienkoAleksandr) [inline packages/plugin-ext/src/plugin/terminal-ext.ts]: `addNewLine` argument was optional. Why Did you change it?

Comment (azatsarynnyy) [reply packages/plugin-ext/src/plugin/terminal-ext.ts]: to declare it as a default parameter which means it's still optional + it provides a default value
